browser/lib/util/webstorage.js

```diff
@@ -1,6 +1,7 @@
 var WebStorage = (function() {
 	var sessionSupported,
 		localSupported,
+        mainClass = (typeof window !== 'undefined' ? window : global),
```

Comment:
I have no issue switching this.   And yeah I agree `globalObject` does make more sense.